src/test/java/net/testudobank/tests/MvcControllerIntegTest.java

```diff
@@ -48,18 +54,19 @@
     .withPassword("db_password")
     .withDatabaseName("testudo_bank");
 
-  
+
   private static MvcController controller;
   private static JdbcTemplate jdbcTemplate;
   private static DatabaseDelegate dbDelegate;
+  private static CryptoPriceClient cryptoPriceClient = Mockito.mock(CryptoPriceClient.class);
```

Comment:
What is the point of mocking this? Why not just use a live `CryptoPriceClient`?

Reply:
Read a bit further. Mocking this makes sense. You are allowing the tester to input the Ethereum Price.

Doesn't this kind of defeat the purpose of an integration/system test if you are giving so much control over things like what an external Crypto Price API returns?